vnext/Shared/Modules/WebSocketModule.cpp

```diff
@@ -185,6 +222,24 @@ void WebSocketTurboModule::Send(string &&message, double forSocketID) noexcept {
 }
 
 void WebSocketTurboModule::SendBinary(string &&base64String, double forSocketID) noexcept {
+  //  VALIDATE Base64 Format - DoS PROTECTION (P0 Critical - CVSS 7.0)
+  try {
+    if (!Microsoft::ReactNative::InputValidation::EncodingValidator::IsValidBase64(base64String)) {
```

Comment:
This looks very bad for perf.  IsValidBase64 looks like it runs a regex against the entire string.  We have plenty of cases where the websocket is sending MB of data.